### PR TITLE
PIMS-1431 Municipality Filter Unit Tests

### PR DESCRIPTION
**Pull request by Fosol**
Also excluding testing classes from test coverage.

** DRAFT PR ** Requires prior PR to be merged first.

### CODE REVIEW

File: backend/dal/Helpers/Extensions/BuildingExtensions.cs [hunk 1]
```diff
@@ -80,6 +80,13 @@ public static IQueryable<Entity.Building> GenerateQuery(this PimsContext context
             if (!String.IsNullOrWhiteSpace(filter.Tenancy))
                 query = query.Where(b => EF.Functions.Like(b.BuildingTenancy, $"%{filter.Tenancy}%"));
 
+            if (!String.IsNullOrWhiteSpace(filter.Municipality))
```
Comment (Fosol) [inline backend/dal/Helpers/Extensions/BuildingExtensions.cs:83]: Can now search for buildings with some parcel properties.

File: backend/entities/Models/BuildingFilter.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@ namespace Pims.Dal.Entities.Models
     /// <summary>
     /// BuildingFilter class, provides a model for filtering building queries.
     /// </summary>
-    public class BuildingFilter : PropertyFilter
+    public class BuildingFilter : ParcelFilter
```
Comment (Fosol) [inline backend/entities/Models/BuildingFilter.cs:10]: Can now search for buildings with some parcel properties.

File: backend/tests/unit/api/Controllers/Admin/ParcelControllerTest.cs [hunk 3]
```diff
@@ -14,13 +14,15 @@
...
 using Pims.Api.Helpers.Exceptions;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Pims.Api.Test.Controllers.Admin
 {
     [Trait("category", "unit")]
     [Trait("category", "api")]
     [Trait("area", "admin")]
     [Trait("group", "parcel")]
+    [ExcludeFromCodeCoverage]
```
Comment (Fosol) [inline backend/tests/unit/api/Controllers/Admin/ParcelControllerTest.cs:25]: Code coverage shouldn't be testing testing code...

File: backend/tests/unit/dal/Services/PropertyServiceTest.cs [hunk 4]
```diff
@@ -54,11 +57,11 @@ public class PropertyServiceTest
                 new object[] { new AllPropertyFilter(50, 24, 50, 26), 0 },
                 new object[] { new AllPropertyFilter() { Agencies = new int[] { 3 } }, 7 },
                 new object[] { new AllPropertyFilter() { ClassificationId = 2 }, 2 },
-                new object[] { new AllPropertyFilter() { Description = "Description" }, 10 },
-                new object[] { new AllPropertyFilter() { Municipality = "Municipality" }, 1 },
+                new object[] { new AllPropertyFilter() { Description = "Description" }, 20 },
```
Comment (devinleighsmith) [inline backend/tests/unit/dal/Services/PropertyServiceTest.cs:60]: what is the significance of the integers here, should they be unique?

Comment (Fosol) [reply backend/tests/unit/dal/Services/PropertyServiceTest.cs:60]: They represent the number of items the filter should return

**Pull request merged**